pkg/agent/clusterclaim.go

```diff
@@ -21,9 +22,11 @@ const (
 	// labelExcludeBackup is true for the local-cluster will not be backed up into velero
 	labelExcludeBackup = "velero.io/exclude-from-backup"
 
-	hypershiftManagementClusterClaimKey   = "hostingcluster.hypershift.openshift.io"
-	hypershiftHostedClusterClaimKey       = "hostedcluster.hypershift.openshift.io"
-	hostedClusterCountFullClusterClaimKey = "hostedclustercount.full.hypershift.openshift.io"
+	hypershiftManagementClusterClaimKey             = "hostingcluster.hypershift.openshift.io"
+	hypershiftHostedClusterClaimKey                 = "hostedcluster.hypershift.openshift.io"
+	hostedClusterCountFullClusterClaimKey           = "hostedclustercount.full.hypershift.openshift.io"
+	hostedClusterCountAboveThresholdClusterClaimKey = "hostedclustercount.above.threshold.hypershift.openshift.io"
+	hostedClusterCountZeroClusterClaimKey           = "hostedclustercount.zero.hypershift.openshift.io"
```

Comment:
really wish the order is xyz.hostedclustercount.hypershift.openshift.io so the different value is in the beginning. Something to ponder about next time 😅

Reply:
very good point